dev_vm/roles/devenv/tasks/main.yml

```diff
@@ -1,27 +1,42 @@
 ---
 - name: Allow root SSH
-  shell: sed -i '/#PermitRootLogin/c\PermitRootLogin yes' /etc/ssh/sshd_config && systemctl restart sshd
+  replace: 
+    path: /etc/ssh/sshd_config
+    regexp: '^#PermitRootLogin yes'
+    replace: 'PermitRootLogin yes'
+    backup: yes
+
+- name: Enable password authentication
+  replace: 
+    path: /etc/ssh/sshd_config
+    regexp: '^PasswordAuthentication no'
+    replace: 'PasswordAuthentication yes'
+    backup: yes
+
+- name: Restart sshd
+  systemd:
+    name: sshd
+    state: restarted
 
 - name: git clone contrail-dev-env
   git:
     repo: 'https://github.com/Juniper/contrail-dev-env.git'
     dest: /root/contrail-dev-env
     update: no
 
-- name: patch startup.sh script to use docker-ce
-  patch:
-    src: startup.patch
-    dest: /root/contrail-dev-env/startup.sh
+- name: Increase docker base size to 200G
```

Comment:
The dev VM size is 40G. I haven't figure out how to increase it. Don't know what would happen if changing docker size 200G here.

Reply:
I doesn't allocate all that disk space, the space is used only when the docker actually has files